### ISSUE TITLE
Using CSS per processor

### ISSUE THREAD

**Issue by rukshn** [lk-gov-health-hiu/covid-19#37]
**Proposal for enhancement**
I think we can have maintainable CSS if we use a CSS pre processor to write CSS instead of plain css.

We will be able to decouple css according to components like buttons, menu, typography etc, or utilities making, padding etc.

And the pre processor will compile them into a single css file, expanded and minified.

We will be able to use the final compiled css file in the project.

This will lead to maintainable css codebase.

**Comment by buddhika75** [comment]
Sure, we will do it.

**Comment by rukshn** [comment]
Yes @diniduhewage i agree should try to use bootstrap and pf as much as possible 

However, there are some style sheet files that are  slowly getting growing. If it continues to grow won't it be difficult to maintain?
